app/alembic/env.py

```diff
@@ -0,0 +1,84 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from models import Base
+
+import os
+from dotenv import load_dotenv, find_dotenv
+load_dotenv(find_dotenv())
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+config.set_main_option('sqlalchemy.url', os.getenv('DB_URL'))
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
```

Comment:
I'm being picky here but to me the 'basically' at the end makes it seem as though whoever wrote this part isn't fully confident in what it does (even though you are).  Just dropping that one word, to me at least, makes you seem much more confident in the code.  Again, being picky.